README.md: added info about snipeit variable

WiktorG351 opened the pull request.

macpijan commented:
I do not like suggestions from this PR. I checked with @johanes2115 today and simply following the README.md worked.

macpijan commented on README.md:
````diff
@@ -161,6 +161,34 @@ Parameters should be defined as follows:
+linking properly. You can try to fix that issue by doing:
+
+```bash
+source ${PWD}/venv/bin/activate
````
This looks quite odd and I would not recommend this

WiktorG351 replied:
removed [here](https://github.com/Dasharo/open-source-firmware-validation/pull/12/commits/263532fb3c2da8abbd4c47df686e810a2fb0920f#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L170)

macpijan commented on README.md:
````diff
@@ -161,6 +161,34 @@ Parameters should be defined as follows:
+Another issue is tests returning:
+
+```bash
+[ ERROR ] Execution stopped by user.
````
This was present at @johanes2115 today, and the reason was submodels were not updated. So not following README was the cause.

WiktorG351 replied:
okay, [removed](https://github.com/Dasharo/open-source-firmware-validation/pull/12/commits/263532fb3c2da8abbd4c47df686e810a2fb0920f#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L176)

macpijan commented on README.md:
````diff
@@ -161,6 +161,34 @@ Parameters should be defined as follows:
+If that doesn't work, consider running this command:
+
+```bash
+git submodule update --init --recursive
````
This is already in the setup section

WiktorG351 replied:
okay, since it's already in the setup section then it shouldn't be here as well, [removed](https://github.com/Dasharo/open-source-firmware-validation/pull/12/commits/263532fb3c2da8abbd4c47df686e810a2fb0920f#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L189)

macpijan commented on README.md:
````diff
@@ -161,6 +161,34 @@ Parameters should be defined as follows:
+it might be useful to run tests with this extra flag:
+
+```bash
+-v snipeit:no
````
This could be documented as a variable, bot rather not in `known issues` section. We can document this variable in usage section.

WiktorG351 replied:
i moved info about `-v snipeit:no` [here](https://github.com/Dasharo/open-source-firmware-validation/pull/12/commits/263532fb3c2da8abbd4c47df686e810a2fb0920f#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L189) and [here](https://github.com/Dasharo/open-source-firmware-validation/pull/12/commits/263532fb3c2da8abbd4c47df686e810a2fb0920f#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R125)

macpijan replied:
This is nice, thanks

WiktorG351 commented:
okay, since following the readme works now I have removed the entire `known issues` section I initially wanted to add. However I would like to add info about the `snipeit` variable, since I think it's useful to know. I have moved that info to the appropriate sections.

macpijan commented:
@WiktorG351 Can we squash, so only the commit adding documentation to README gets merged?

Can we also rename it to foloow typical convention: `README.md: add ...`

WiktorG351 commented:
@macpijan hi, i squashed the two commits and changed the title

macpijan commented:
Thanks, LGTM.
